### PR TITLE
Fix: 819 Missing Bitcoin/BCH URI in Receive Money & Listing Payment

### PR DESCRIPTION
**Pull request by jashot7**
Link to Issue: https://github.com/OpenBazaar/openbazaar-desktop/issues/819

I kept a light touch on this intentionally. Tested thoroughly to ensure TBTC/TBCH wallet address can be properly copied/pasted and that the QR code is generated correctly with the new URI format. 

Let me know If you see any issues or request that I do anything different from how it was done.

### CODE REVIEW

File: js/views/modals/purchase/Payment.js [hunk 1]
```diff
@@ -124,10 +124,11 @@ export default class extends BaseVw {
     this.getCachedEl('.js-payFromWallet').addClass('processing');
     this.spendConfirmBox.setState({ show: false });
     const currency = getServerCurrency().code;
+    const addressNoPrefix = this.paymentAddress.split(':')[1];
```
[the remark this answers is not in the document]
Comment (jashot7) [reply js/views/modals/purchase/Payment.js]: Yeah, it sort of looks that way but these changes ensure that we're using the URI format where necessary and the old non URI format for our server calls where necessary.

[the remark this answers is not in the document]
Comment (jashot7) [reply js/views/modals/purchase/Payment.js]: I'll note again: I tested this as thoroughly as I could for TBTC and TBCH: Receiving money & funding a listing: the QR code translates appropriately, the text copies appropriately and funds receive/send successfully.

[the remark this answers is not in the document]
Comment (jashot7) [reply js/views/modals/purchase/Payment.js]: Previously the text copied/qr code translated didn't include the URI format and currently (in this PR) it now does.

**Comment by jjeffryes** [comment]
@rmisio this looks good to me, and works as intended when I test it. Did you want to do a review too?

**Comment by rmisio** [comment]
Turns out having the `bitcoin:` prefix is not part of Bitcoin's address spec. Consequently, we will not be implementing to ensure our address format plays nice with all spec adhering wallets.

**Pull request closed without merging**